packages/charting/controller/src/index.js

```diff
@@ -55,18 +56,25 @@ export const getScore = (question, session, env = {}) => {
     let score = 0;
 
     const scoreForLabelAndValueEditable = (answer, corrAnswer) => {
-      const { value, label } = answer;
-
+      const { value, label, index } = answer;
+      let firstEquality = false;
+      let secondEquality = false;
       maxScore += 2;
 
       if (value === corrAnswer.value) {
```

Comment:
It's a little bit confusing what firstEquality and secondEquality mean.

I assume:
firstEquality - if value is correct
secondEquality - if label is also correct
Is my assumption correct? If yes, can we maybe rename the variables, please?

Maybe we can store those values in variables and refactor where needed.
Something ike
`valueIsCorrect = value === corrAnswer.value`
`labelIsCorrect = checkLabelsEquality(label, corrAnswer.label)`

Reply:
Done